apps/passport-client/components/screens/FrogScreens/ScoreTab.tsx

```diff
@@ -23,17 +23,21 @@ export function ScoreTab({ score }: { score?: FrogCryptoScore }) {
   return (
     <Container>
       <ScoreTable title="You" scores={[score]} />
-      {scores.length > 0 && <ScoreTable title="Leaderboard" scores={scores} />}
+      {scores.length > 0 && (
+        <ScoreTable title="Leaderboard" scores={scores} myScore={score} />
+      )}
     </Container>
   );
 }
 
 function ScoreTable({
   title,
-  scores
+  scores,
+  myScore
 }: {
   title: string;
   scores: FrogCryptoScore[];
+  myScore?: FrogCryptoScore;
```

Comment:
why is this param optional?

Reply:
for the top view of my score, I don't want it to bold so didn't pass it in

<img width="357" alt="image" src="https://github.com/proofcarryingdata/zupass/assets/4317392/5fa57048-00d3-4d13-be62-d987a9ec02a2">